src/NumericalAlgorithms/Spectral/SwshCoefficients.cpp

```diff
@@ -23,33 +22,33 @@ Coefficients::Coefficients(const size_t l_max) noexcept : l_max_(l_max) {
   alm_info_.reset(alm_to_initialize);
 }
 
+namespace {
+template <size_t I>
+const Coefficients& coefficients_cache_impl() noexcept {
+  static const Coefficients precomputed_coefficients{I};
+  return precomputed_coefficients;
+}
+
 template <size_t... Is>
 SPECTRE_ALWAYS_INLINE const Coefficients& precomputed_static_coefficients_impl(
     const size_t index, std::index_sequence<Is...> /*meta*/) noexcept {
   if (UNLIKELY(index > collocation_maximum_l_max)) {
     ERROR("The provided l_max "
           << index
           << " is not below the maximum l_max to cache, which is currently "
-          << coefficients_maximum_l_max
-          << ". Either construct the Coefficients manually, or consider (with "
-             "caution) increasing the `cooefficients_maximum_l_max`.");
+          << collocation_maximum_l_max
+          << ". Either "
+             "construct the Collocation manually, or consider (with caution) "
```

Comment:
was the replacement of Coefficients by Collocation intentional?  I'm not sure I understand what this error message is telling me to do if I don't want to increase coefficients_maximum_l_max